Dashboard/app/js/lib/components/devices/AssignmentsEditView/screens/EditDevices.jsx

```diff
@@ -16,9 +16,7 @@ export default class EditDevices extends React.Component {
     // filter out selected devices
     const { devices, selectedDeviceIds } = this.context.data;
 
-    const filteredDevices = devices.filter(device =>
-      selectedDeviceIds.includes(device.id)
-    );
+    const filteredDevices = devices.filter(device => selectedDeviceIds.includes(device.id));
```

Comment:
Did you merge the code from the previous PR? I thought this one had changed...
```suggestion
    const selectedDevices = devices.filter(device => selectedDeviceIds.includes(device.id));
```

Reply:
This is different